### DISCUSSION TITLE
Different runtime for 2 different rules

### DISCUSSION THREAD

**Discussion by ngms17** [jertel/elastalert2#834] [Q&A] [answered]
Hi,

I want to configure elastalert to use 2 different rules. One of them, i would like to run every minute while the other one i would like to run every 15 min (for exemple). 

The "run_every" option on elastalert config file seems unique for every single rule that is configured. Is it possible to have 2 different rules running in 2 different spaces of time?

**Comment by jertel** [comment, marked as answer]
`run_every` should be configurable per rule, but it's not a usage scenario I've used, so it's possible there is a problem with it. Have you tried it?

If it's not behaving as expected, you could use the [limit_execution](https://elastalert2.readthedocs.io/en/latest/ruletypes.html#limit-execution) defined in the rule, as another method to reduce executions of certain rules.

**Reply by ngms17** [reply]
Hi.

I haven´t tried yet. If `run_every` should be configurable per rule, this means that i can comment out the `run_every` option on `config.yaml`?

**Reply by jertel** [reply]
It's best to leave that one in there, as the default. Then, only override the value in the rules that need to have a custom execution interval.
